src/main/resources/data/computercraft/lua/rom/apis/rednet.lua

```diff
@@ -26,8 +26,8 @@ CHANNEL_BROADCAST = 65535
 CHANNEL_REPEAT = 65533
 
 local tReceivedMessages = {}
-local tReceivedMessageTimeouts = {}
 local tHostnames = {}
+local nClearTimer
```

Comment:
*cries about Hungrarian notation*. Probably not worth doing the cleanup right now. Soon :tm:.

Reply:
Yeah, as much as I'd like to use more normal variable naming, I'm leaving it as-is to keep it consistent. At least consistently bad is better than inconsistent?